src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue

```diff
@@ -1,142 +1,79 @@
 <template>
-	<div :class="{ 'overlay-shown': isUserInExp }">
-		<div class="sitewide-appeal-wrapper">
-			<div class="sitewide-appeal row">
-				<div class="sitewide-header small-12 medium-9 medium-offset-2 large-9 large-offset-2 columns">
-					<h2 class="strong" v-html="bannerHeadline"></h2>
-					<fifteen-years-button>Button</fifteen-years-button>
+	<fifteen-years-styles class="appeal-15-wrapper">
```

Comment:
I don't think you'll want this here, since it changes the max-width of .row class. 
This means on the homepage you'll have a different max-width than the content below.
![image](https://user-images.githubusercontent.com/187937/94322254-c8cea180-ff46-11ea-9f46-fb444b2908c4.png)

Instead I'd recommend applying mixins you need from the sass import you have `@import 'components/15-years/15-years'` to get your headers or whatever to the 15-year look.

Reply:
👍